test/integration/VisualizarCards.feature

```diff
@@ -0,0 +1,107 @@
+#language: pt
+Funcionalidade: Visualizar testes dos experimentos como cards
+  Eu como um usuário
+  Eu quero visualizar os testes como cards
+
+Contexto:
+  Dado que estou logado como usuário do ide-experimentador
+  E eu estou na página do experimento
+
+# Cenários Gerais (relacionados a lista de testes)
+# Cenário triste
+Cenário: Não há testes no experimento

```

Comment:
Esse não é um sad path, visto que é esperado que uma mensagem de que não há testes seja mostrada se realmente não há testes cadastrados. Pensem no sad path como uma situação que vocês usariam try and catch, por exemplo.